index.js

```diff
@@ -173,22 +184,20 @@ export default e => {
             }
             {
               const collisionId = collision.objectId;
-              const object = getAppByPhysicsId(collisionId);
-              if (object) {
-                if (!object.arrowApps) {
-                  object.arrowApps = [];
+              const targetApp = getAppByPhysicsId(collisionId);
+              if (targetApp) {
+                if (!targetApp.arrowApps) {
+                  targetApp.arrowApps = [];
```

Comment:
We should not decorate other apps like this, since it can interfere.

If you need a map, use a local `Map` in a closure.